docs/topics/db/examples/proxy_models.txt

```diff
@@ -0,0 +1,93 @@
+===========================
+Proxy model as relationship
+===========================
+
+.. highlight:: pycon
```

Comment:
this might be python?

Reply:
I took that from another file, looks like it should be `python` indeed, I'll open a separate PR to fix the other instances.

```suggestion
.. highlight:: python
```

Reply:
See: https://github.com/django/django/pull/11069